pom.xml

```diff
@@ -69,8 +69,8 @@
   </scm>
 
   <properties>
-    <jenkins.version>2.46.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.106</jenkins.version>
```

Comment:
I doubt it can be considered for merge before it hist LTS

Reply:
A topic for @rtyler and Evergreen I suppose.